src/features/game/types/removeables.ts

```diff
@@ -275,6 +276,14 @@ export const REMOVAL_RESTRICTIONS: Partial<
   "Heart of Davy Jones": (game) => areAnyTreasureHolesDug(game),
   "Maneki Neko": (game) => hasShakenManeki(game),
   "Time Warp Totem": (_: GameState) => [true, "In use"],
+
+  // Fishing Boosts
+  Alba: (game) => {
```

Comment:
We should always use the sstrict equality ( === ) operator in any kind of conditional logic because the way javascript performs type coercion in cases where you don't expect.

Also, I don't really understand this remove condition? Alba is only in use if you haven't used it for the day?

Reply:
@elias7905 Proposed that "fished today" should be the criteria for locking.  I'm open to other ideas, if that doesn't make sense to you.

Reply:
> We should always use the sstrict equality ( === ) operator

Where is this referring to?  Thanks.

Reply:
> Alba is only in use if you haven't used it for the day?

Ah, I see what you're saying now.  Typo: should have been `!==`.